src/frontend/js/widgets/Dashboard/components/TeacherOrganizationDashboardSidebar/index.tsx

```diff
@@ -7,20 +7,37 @@ import {
   getDashboardRouteLabel,
   getDashboardRoutePath,
 } from 'widgets/Dashboard/utils/dashboardRoutes';
+import { useOrganization } from 'hooks/useOrganizations';
+import { Spinner } from 'components/Spinner';
+import { DashboardAvatar, DashboardAvatarVariantEnum } from '../DashboardAvatar';
 
 const messages = defineMessages({
+  header: {
+    id: 'components.TeacherOrganizationDashboardSidebar.header',
+    description: 'The title of the organization dashboard sidebar',
+    defaultMessage: '{organizationTitle}',
+  },
```

Comment:
Useless nope ?

Reply:
we have it in other pages.
i imagine that allow us to translate it into "Your are in {organizationTitle}" if needed.
I can remove it.

Reply:
Yup remove it please. From my point of view, I don't think this is the purpose of translation messages to allow to customize UI.

Reply:
I already removed it.
For me it's not customizing UI, it's giving to the translation a variable to use when choosing the wording.